Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp

```diff
@@ -564,4 +571,24 @@ namespace AssetProcessor
             }
         }
     }
+
+    void BuilderManager::PrintDebugOutput()
+    {
+        // If debug output was tracked, print it on shutdown.
+        // This prints each asset that was processed by each builder, in the order they were processed.
+        // This is useful for tracing issues like memory leaks across assets processed by the same builder.
+        for (auto builderInfo : m_builderDebugOutput)
+        {
+            AZ_TracePrintf("BuilderManager", "Builder %.*s processed these assets:\n",
```

Comment:
You can use AZ_STRING_FORMAT instead of %.*s here

Reply:
We talked about this. I prefer it as-is, I find AZ_STRING_FORMAT is much longer and makes these string formats a bit harder to read than %.*s, so I'm keeping it as-is.